app/policies/level_b_policy.rb

```diff
@@ -0,0 +1,5 @@
+class LevelBPolicy < ApplicationPolicy
```

Comment:
Do you think this would be a place for the level b activity upload policy to live? If so, shall we add a new commit to this PR for refactoring that bit of functionality? Or at least add a technical debt card for it (though it'd be nice to sort rather than add to the pile!)

Reply:
Done - I started writing an `activity_upload` method in the policy, but then changed tack, per the commit message

Reply:
As discussed on Slack, we'll do what you originally did (an `activity_upload` method) as it'll allow us to be a bit more granular, just in case we want to split permissions for the different entities being uploaded in the future e.g. a finance team may be responsible for uploading budgets.